netmanager/src/views/pages/Map/OverlayMap.js

```diff
@@ -127,7 +118,7 @@ const PollutantSelector = ({ className, onChange, showHeatMap }) => {
   useInitScrollTop();
   const orgData = useOrgData();
   const [open, setOpen] = useState(false);
-  const [pollutant, setPollutant] = useState("pm2_5");
+  let pollutant = localStorage.pollutant;
```

Comment:
Pollutant value is stored and fetched from localstorage.

Reply:
@Codebmk When is this pollutant value removed?  Or does it stay in the client's localstorage forever?

Reply:
@FaithDaka I have addressed this

Reply:
A quick one @Codebmk , this deletion of items in the local storage, wasn't this functionality already present whenever a user logs out of the platform?

Thanks for the works.

Reply:
Yes it is, but for every new item added, you need to explicitly define those items to remove them